File: 1637-widest-vertical-area-between-two-points-containing-no-points/1637-widest-vertical-area-between-two-points-containing-no-points.cpp
class Solution {
public:
    int maxWidthOfVerticalArea(vector<vector<int>>& points) {
        vector <int> x(points.size());
        
        for (int i = 0;i < points.size(); i++){
            x[i] = points[i][0];
        }
        
        sort (x.begin() , x.end());
        int maxDiff {};
        
        for (int i = 1; i < x.size(); i++)
            maxDiff = max (maxDiff , x[i] - x[i - 1]);
        
        
        return maxDiff;
    }
};

File: 1637-widest-vertical-area-between-two-points-containing-no-points/1637-widest-vertical-area-between-two-points-containing-no-points.cs
public class Solution {
    public int MaxWidthOfVerticalArea(int[][] points) {
        int n = points.Length;
        int [] x = new int [n];
        
        for (int i = 0;i < n; i++){
            x[i] = points[i][0];
        }
        
        Array.Sort (x);
        
        int maxDiff = 0;
        
        for (int i = 1; i < n; i++)
            maxDiff = Math.Max (maxDiff , x[i] - x[i - 1]);
        
        
        return maxDiff;
    }
}